cli/dcoscli/node/main.py

```diff
@@ -498,7 +498,8 @@ def _log(follow, lines, leader, slave, component, filters):
     lines = util.parse_int(lines)
 
     # if journald logging is disabled. Read from files API and exit.
-    if not log.dcos_log_enabled():
+    # https://github.com/dcos/dcos/blob/master/gen/calc.py#L151
+    if 'journald' not in log.logging_strategy():
```

Comment:
no this could be either `journald` or `journald+logrotate` we need to run this if no journald in string found